packages/plugin-commands-init/package.json

```diff
@@ -40,6 +40,8 @@
     "@pnpm/config": "workspace:13.13.2",
     "@pnpm/error": "workspace:2.1.0",
     "@pnpm/write-project-manifest": "workspace:2.0.11",
+    "camelcase-keys": "^6.2.2",
+    "read-ini-file": "^3.1.0",
```

Comment:
all the settings from `.npmrc` are already passed to the plugin via the `rawConfig` option.

Reply:
I have modified it, please check whether it is correct @zkochan